about.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://cdn.tailwindcss.com"></script>
+  <script src="https://kit.fontawesome.com/b45a87b3cb.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header>
+    <div class="hidden md:flex gap-5 bg-[#272a31] justify-end pr-[12.5vw]">
+      <div class="flex gap-2 py-3 text-white">
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+      </div>
+      <div class="flex gap-5 py-2">
+        <p class="text-white">English</p>
+        <p class="text-white">My Page</p>
+        <p class="text-white">Logout</p>
+      </div>
+    </div>
+    <nav class="hidden bg-white md:flex justify-around shadow-xl">
+      <div>
+        <a href="./index.html"><img class="h-20" src="./Assets/4117741.svg"></a>
```

Comment:
- Please make sure the image contains `alt` attributes

Reply:
Thanks for the suggestions I have fixed the issue